rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -62,245 +49,62 @@ class Airdrop(NamedTuple):
-        'https://gist.githubusercontent.com/LefterisJP/727c05f3a9cab79059258528595c102e/raw/59ff367bd532f632ce94e69900709155a55de82a/poap_ycover.json',
-        'https://poap.delivery/ycover',
-        'A New Face For yCover',
-    ),
-    'yfi_og': (
-        # is checksummed
-        'https://gist.githubusercontent.com/LefterisJP/58862ec2b398c770d60c24e50e18e50c/raw/3292995f530baedcdacde02526c6b2bd1de0e110/poap_yfi_og.json',
-        'https://poap.delivery/yfi-og',
-        'I Played 4 YFI',
-    ),
-}
+    May raise - RemoteError if the request fails or metadata is invalid.
+    """
+    try:
+        response = requests.get(url=AIRDROPS_INDEX, timeout=CachedSettings().get_timeout_tuple()).json()  # noqa: E501
+        airdrops: dict[str, Airdrop] = {}
+        for protocol_name, airdrop_data in response['airdrops'].items():
+            if 'new_asset_data' in airdrop_data:
+                if AssetType(airdrop_data['new_asset_data']['asset_type']) == AssetType.EVM_TOKEN:
+                    asset = airdrop_data['asset'] = get_or_create_evm_token(
+                        userdb=database,
+                        evm_address=airdrop_data['asset_identifier'].split(':')[-1],
+                        chain_id=ChainID.deserialize(airdrop_data['new_asset_data']['chain_id']),
+                        decimals=airdrop_data['new_asset_data']['decimals'],
+                        name=airdrop_data['new_asset_data']['name'],
+                        symbol=airdrop_data['new_asset_data']['symbol'],
+                    )
+                else:
+                    GlobalDBHandler.add_asset(CryptoAsset.initialize(
+                        identifier=airdrop_data['asset_identifier'],
+                        asset_type=AssetType(airdrop_data['new_asset_data']['asset_type']),
+                        name=airdrop_data['new_asset_data']['name'],
+                        symbol=airdrop_data['new_asset_data']['symbol'],
+                        coingecko=airdrop_data['new_asset_data'].get(['coingecko'], None),
+                        cryptocompare=airdrop_data['new_asset_data'].get(['cryptocompare'], None),
+                    ))
+            else:
+                asset = Asset(airdrop_data['asset_identifier']).resolve_to_evm_token()
+            airdrop_data['asset'] = asset
+            airdrops[protocol_name] = Airdrop(**{
+                field: airdrop_data[field] for field in airdrop_data
+                if field in Airdrop._fields
+            })
+        poap_airdrops: dict[str, list[str]] = response['poap_airdrops']
+    except requests.exceptions.RequestException as e:
+        raise RemoteError(f'Airdrops Index request failed due to {e!s}') from e
+    except KeyError as e:
+        raise RemoteError(f'Airdrops Index contains an invalid JSON {e!s}') from e
+    except DeserializationError as e:
+        raise RemoteError(f'Airdrops Index contains an invalid ChainID of a token. {e!s}') from e
+    except UnknownAsset as e:
```

Comment:
what error can you raise from

```
AssetType(airdrop_data['new_asset_data']['asset_type'])
```

I beliegve it is valueError? It should have a deserialize method too